feature-libs/user/account/components/login-form/login-form.module.ts

```diff
@@ -49,7 +50,7 @@ import { LoginFormComponent } from './login-form.component';
       cmsComponents: {
         ReturningCustomerLoginComponent: {
           component: LoginFormComponent,
-          guards: [NotAuthGuard],
+          guards: [NotAuthGuard, CustomLoginGuard],
```

Comment:
`query:` do we want to run this guard even for password flow? I couldn't found any toggle/configuration that hides the csrf call which is redundant for password flow and may cause issues. Apologies if I'm missing something obvious.

Reply:
I added a flag to the AuthConfigService to indicate when the feature is enabled, and used it to short-circuit the guard when the feature is disabled.  This will allow ROPC grant flow to function with appropriate configuration.